pkg/controller/database/dbsubnetgroup/controller.go

```diff
@@ -129,21 +130,21 @@ func (e *external) Observe(ctx context.Context, mgd resource.Managed) (managed.E
 }
```

Comment:
In the `ExternalObservation` above I believe we are going to always be calling for an `Update` because `ResourceUpToDate` will default to `false` (ref: https://github.com/crossplane/provider-aws/blob/a7a3b6ba7d29821a7e84e304f043ec462a1ceb8a/pkg/controller/database/rdsinstance.go#L160). This was fine before because we had a no-op update, but now it looks as though we are performing actual updates, which we won't want to do unless they are needed. We will need to compare the observed DB subnet group with the spec and determine if we need an update, then provide the appropriate value to `ResourceUpToDate`. Let me know if you have any questions :)